test/games/strategy/engine/data/TestDelegateBridge.java

```diff
@@ -30,7 +31,7 @@
   private PlayerID m_id;
   private String m_stepName = "no name specified";
   private IDisplay m_dummyDisplay;
-  private final DummySoundChannel m_soundChannel = new DummySoundChannel();
+  private final ISound m_soundChannel = mock(ISound.class);
```

Comment:
`TestDelegateBridge` should also be a mock. You may wind up definding something that looks like this:
```
TestDelegateBridge mockBridge
ISound mockSound;
:
:
expect(mockBridge.getSound().andReturn(mocksound);
expect(mockSound.doSomething()).andReturn(something);
```